.github/workflows/format.yaml

```diff
@@ -57,8 +57,11 @@ jobs:
     - name: Installing formatting tools
       run: |
         sudo apt-get update
-        sudo apt-get install clang-format
+        sudo apt-get install clang-format-10
         pip install pyformat
+        # Use clang-format 10 as default until we choose to migrate to 11
+        sudo update-alternatives --remove-all clang
```

Comment:
I think you should either remove clang-format only or install the clang/clang++ links as well. Not sure whether clang-format calls clang or not.